app/controllers/application_controller/tree_support.rb

```diff
@@ -20,6 +20,11 @@ def squash_toggle
     end
   end
 
+  def find_record
+    identify_record(params[:id], controller_name.camelize) if %w(container_image host).include? controller_name
+    identify_record(params[:id], VmOrTemplate)
```

Comment:
I wish we could do [model.base_class](https://github.com/ManageIQ/manageiq/blob/25cba3d51de9461f729b73b9b0a13a1f3b728911/app/controllers/application_controller.rb#L71), but I don't really know why we can't.